### PR TITLE
tikv-ctl: improve scan and fix a bug for mvcc

### PR DESCRIPTION
**Pull request by hicqu**
Use tikv-ctl scan with bad arguments maybe cause TiKV crash. This PR fix this.

### CODE REVIEW

File: src/bin/tikv-ctl.rs [hunk 1]
```diff
@@ -824,20 +828,23 @@ fn main() {
...
                     Arg::with_name("from")
+                        .required(true)
                         .short("f")
                         .long("from")
                         .takes_value(true)
                         .help("set the scan from raw key, in escaped format"),
                 )
                 .arg(
                     Arg::with_name("to")
+                        .conflicts_with("limit")
```
Comment (siddontang) [inline src/bin/tikv-ctl.rs]: why `limit` conflicts with `to`?  I think getting limited items in `[from, to)` is reasonable.

**Comment by hicqu** [comment]
PTAL @overvenus .

File: src/bin/tikv-ctl.rs [hunk 2]
```diff
@@ -778,7 +791,7 @@ fn main() {
                                 .conflicts_with_all(&["region", "index"])
                                 .short("k")
                                 .takes_value(true)
-                                .help("set the raw key, in escaped form"),
+                                .help("set the raw key (generally starts with \"z\"), in escaped form"),
```
Comment (overvenus) [inline src/bin/tikv-ctl.rs]: How about replacing with a constant string?

**Comment by overvenus** [comment]
LGTM

File: src/bin/tikv-ctl.rs [hunk 3]
```diff
@@ -855,8 +871,8 @@ fn main() {
                         .help("set the scan commit_ts as filter"),
                 )
                 .arg(
-                    Arg::with_name("cf")
-                        .long("cf")
+                    Arg::with_name("show-cf")
```
[the remark this answers is not in the document]
Comment (hicqu) [reply src/bin/tikv-ctl.rs]: `cf` is unclear. People will think the command `scan` on `cf`s they specified. But the command `scan` on `write` cf and print specified `cf`s.

[the remark this answers is not in the document]
Comment (breeswish) [reply src/bin/tikv-ctl.rs]: ok

**Comment by breeswish** [comment]
LGTM

**Pull request merged**